### ISSUE TITLE
docker crash on start

### ISSUE THREAD

**Issue by zhaodc** [docker/for-mac#2432]
```
Docker for Mac: version: 17.12.0-ce-mac47 (72b93a017350990850ddc37cd341bd16fce3e911)
macOS: version 10.13.2 (build: 17C88)
logs: /tmp/2954FA03-5AE4-4BB5-94F2-2721B2884FF7/20180115-122930.tar.gz
failure: com.docker.vpnkit is not running
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/zhaodc/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/zhaodc/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/zhaodc/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         slirp check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/zhaodc/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[OK]     disk
```

**Comment by akimd** [comment]
Could you please see in https://github.com/docker/for-mac/issues/2383#issuecomment-358246040 if that would solve your issue?

**Comment by gtardif** [comment]
If you get a chance, can you uninstall and then try this test build: https://download-stage.docker.com/mac/pr/21935/Docker.dmg. 
Keep in mind this is a test build. Ignore any update it might propose, and after the test you will have to reinstall an Edge or Stable version to get proper updates again. 
Let us know if this works (or upload diagnostics if not) ; if this works fine we'll integrate this fix in a proper release soon.

**Comment by akimd** [comment]
Could you please check if the following image fixes your problems?  You might need to reset to factory defaults...

https://download-stage.docker.com/mac/pr/21949/Docker.dmg

This is a test build. Ignore any update it might propose, and after testing, reinstall an Edge or Stable version to get proper updates again. Thanks!

**Comment by gtardif** [comment]
We just released 17.12.0-ce-mac49 on the Stable channel that contains the fix mentioned above. (and 18.01 on the Edge channel should also fix this)
